view/frontend/web/js/delete-certificate.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * ClassyLlama_AvaTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2018 Avalara, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+require(['jquery', 'mage/translate', 'Magento_Ui/js/modal/confirm'], function ($, $t, confirm) {
+    'use strict';
+
+    $('.delete-button').click(function(e) {
```

Comment:
Never use selectors if at all possible to avoid. Here you could data-mage-init on the element, passing it the url to use as an argument to the component. This will give you an element reference.